ts/features/itwallet/common/components/ItwEngagementBanner.tsx

```diff
@@ -0,0 +1,148 @@
+  description: string;
+  action: string;
+  dismissable?: boolean;
+  // Events
+  onPress: () => void;
+  onDismiss: () => void;
+};
+
+export const ItwEngagementBanner = ({
+  testID,
+  title,
+  description,
+  action,
+  dismissable,
+  onPress,
+  onDismiss
+}: WithTestID<Props>) => {
+  const handleOnClosePress = useCallback(() => {
+    Alert.alert(
+      I18n.t("features.itWallet.engagementBanner.dismissAlert.title"),
+      I18n.t("features.itWallet.engagementBanner.dismissAlert.description"),
+      [
+        {
+          text: I18n.t(
+            "features.itWallet.engagementBanner.dismissAlert.cancel"
+          ),
+          onPress: constNull
+        },
+        {
+          text: I18n.t(
+            "features.itWallet.engagementBanner.dismissAlert.confirm"
+          ),
+          style: "destructive",
+          onPress: onDismiss
+        }
+      ]
+    );
+  }, [onDismiss]);
+
+  // Generates a complete fallbackAccessibilityLabel by concatenating the title, content, and action
+  // if they are present. Removes markdown formatting characters like asterisks.
+  const accessibilityLabel = [title, description, action]
+    .filter(Boolean)
+    .join("\n")
+    .replace(/\*/g, "");
+
+  const markdownRules = useMemo(
+    () => ({
+      Paragraph(paragraph: TxtParagraphNode, render: Renderer) {
+        return (
+          <BodySmall key={getTxtNodeKey(paragraph)} weight="Regular">
+            {paragraph.children.map(render)}
+          </BodySmall>
+        );
+      },
+      Strong(strong: TxtStrongNode, render: Renderer) {
+        return (
+          <BodySmall key={getTxtNodeKey(strong)} weight="Semibold">
+            {strong.children.map(render)}
+          </BodySmall>
+        );
+      }
+    }),
+    []
+  );
+
+  return (
+    <View
+      testID={testID}
+      accessible={true}
+      // A11y related props
+      accessibilityLabel={accessibilityLabel}
+      accessibilityRole={"button"}
+      onAccessibilityTap={onPress}
+    >
+      <ItwBrandedBox borderRadius={8}>
+        <ItWalletDeck width={105} height={145} style={styles.deck} />
+        <VStack space={8}>
+          <View style={styles.header}>
+            <H4 color="black" style={styles.content}>
+              {title}
+            </H4>
+            {dismissable && (
+              <IconButton
+                testID="itwEngagementBannerCloseButtonTestID"
+                color="contrast"
+                accessibilityLabel="close"
+                icon="closeMedium"
+                onPress={handleOnClosePress}
+              />
+            )}
+          </View>
+          <View style={styles.content}>
+            <IOMarkdown rules={markdownRules} content={description} />
+          </View>
+          <PoweredByItWalletText />
+        </VStack>
+        <VSpacer size={8} />
+        <IOButton
+          testID="itwEngagementBannerActionButtonTestID"
+          color="primary"
+          label={action}
+          onPress={onPress}
```

Comment:
On Android pressing on this button does nothing. It seems the problem is within `ItwBrandedBox`: if I remove it the button works again.